src/meta/src/barrier/mod.rs

```diff
@@ -249,10 +260,6 @@ where
     fn post_resolve(&mut self, command: &Command) {
         match command.changes() {
             CommandChanges::DropTables(tables) => {
-                assert!(
```

Comment:
Since we have `remove_changes` in `cancel_command`, this assertion should still be able to hold?

Reply:
I see... So we first call `post_resolve` and then call `cancel_command` in barrier completion?

Reply:
Yes exactly.